internal/controllers/machine/suite_test.go

```diff
@@ -97,11 +97,12 @@ func TestMain(m *testing.M) {
 			SetConnectionCreationRetryInterval(2 * time.Second)
 
 		if err := (&Reconciler{
-			Client:                      mgr.GetClient(),
-			APIReader:                   mgr.GetAPIReader(),
-			ClusterCache:                clusterCache,
-			RemoteConditionsGracePeriod: 5 * time.Minute,
-			AdditionalSyncMachineLabels: nil,
+			Client:                           mgr.GetClient(),
+			APIReader:                        mgr.GetAPIReader(),
+			ClusterCache:                     clusterCache,
+			RemoteConditionsGracePeriod:      5 * time.Minute,
+			AdditionalSyncMachineLabels:      nil,
+			AdditionalSyncMachineAnnotations: nil,
```

Comment:
If these are regex patterns and won't change until restarting controller, I was wondering if we could pre-compile and store them in advance instead of storing pattern strings and compile every time in the reconciler.